modules/identities/module.json

```diff
@@ -0,0 +1,64 @@
+{
+    "name": "Identites",
+    "description": "Manage identities and identity data for users.",
+    "icon": "key",
+    "tags": [
+        "core",
+        "user",
+        "auth"
+    ],
+    "authors": [
+        "rivet-gg",
+        "Blckbrry-Pi"
+    ],
+    "status": "beta",
+    "dependencies": {
+        "rate_limit": {},
+        "users": {},
+        "tokens": {}
+    },
+    "scripts": {
```

Comment:
Indicate that these scripts should be called by the identity provider. They're already internal, but it's very likely a dev will try to figure out how to sign in and try to use the wrong internal endpoint.

Reply:
How should I do that? Description, name, or both?

Reply:
How should I do that? Description, name, or both?

Reply:
actually we could add this to the module description instead of the script descriptions. something like "intended for internal use by modules exposing auth providers."